Core/src/main/java/com/plotsquared/core/util/PlayerManager.java

```diff
@@ -164,25 +164,51 @@ public static void getUUIDsFromString(
      * @return The player's name, None, Everyone or Unknown
      */
     public static @NonNull String getName(final @Nullable UUID owner) {
-        return getName(owner, true);
+        return getName(owner, LocaleHolder.console());
```

Comment:
Might be best to default to null here and deprecate the method, or state that no messages will be sent in the JD comments above. If someone is accessing the method from another plugin and suddenly start getting console output from it, it'll be a little confusing

Reply:
Yeah, deprecating the current methods and introduce new ones are the better way to go.